external/dir.props

```diff
@@ -0,0 +1,13 @@
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <!-- HACK to work around binplacing issue right now for depproj's -->
+    <IsRuntimeAssembly>false</IsRuntimeAssembly>
+  </PropertyGroup>
+  <PropertyGroup>
```

Comment:
Should this be conditioned on Exists('$(MSBuildThisProjectDirectory)project.json.template') AND !Exists('$(ProjectJson)')